vulkan/vulkan_interface.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_VULKAN_VULKAN_INTERFACE_H_
+#define FLUTTER_VULKAN_VULKAN_INTERFACE_H_
+
+#define VK_NO_PROTOTYPES 1
+
+#include "third_party/vulkan/src/vulkan/vulkan.h"
+
+#define VK_KHR_ANDROID_SURFACE_EXTENSION_NAME "VK_KHR_android_surface"
```

Comment:
That entire section can be replaced with `define VK_USE_PLATFORM_ANDROID_KHR 1` on Android. Unfortunately, that also pulls in Android specific headers from the NDK. So I put that in there as a temporary workaround to get completions working. Will remove the whole thing in an upcoming patch.